src/app/core/core.module.ts

```diff
@@ -92,7 +94,11 @@ export function getConfiguration(configService: ConfigService) {
 			deps: [ConfigService],
 			multi: true
 		},
-		{ provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor, multi: true }
+		[
+			{ provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor, multi: true },
```

Comment:
@sshah-asymmetrik what guarantees the order of interceptors if we're decorating the error with an attribute that needs to be respected by subsequent interceptors?

Reply:
@brianghig  Angular itself should be guaranteeing that order. Interceptors are executed in the order that they're provided, so in this case outgoing requests will go AuthInterceptor -> EuaAuthInterceptor -> SigninAuthInterceptor (although we don't do anything to them on their way out) and incoming responses will handled SigninAuthInterceptor -> EuaAuthInterceptor -> AuthInterceptor.

I've verified that this is the behavior, and tested it with additional interceptors in an application module, where it appears to just append the additional interceptors to the list found here in the core. If interceptors are applied in several different modules, it may take some fanciness to set the order correctly.